test/datasets/ljspeech_test.py

```diff
@@ -0,0 +1,80 @@
+import csv
+import os
+
+from torchaudio.datasets import ljspeech
+
+from ..common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    normalize_wav,
+    save_wav,
+)
+
+
+class TestLJSpeech(TempDirMixin, TorchaudioTestCase):
+    backend = "default"
+
+    root_dir = None
+    data = []
+    transcripts = [
+        "Test transcript 1",
+        "Test transcript 2",
+        "Test transcript 3",
+        (
+            "The examination and testimony of the experts enabled the "
+            "Commission to conclude that 5 shots may have been fired,"
+        ),
+    ]
+
+    normalized_transcripts = [
+        "Test transcript one",
+        "Test transcript two",
+        "Test transcript three",
+        (
+            "The examination and testimony of the experts enabled the "
+            "Commission to conclude that five shots may have been fired,"
+        ),
+    ]
+
+    @classmethod
+    def setUpClass(cls):
+        cls.root_dir = cls.get_base_temp_dir()
+        base_dir = os.path.join(cls.root_dir, "LJSpeech-1.1")
+        archive_dir = os.path.join(base_dir, "wavs")
+        os.makedirs(archive_dir, exist_ok=True)
+        metadata_path = os.path.join(base_dir, "metadata.csv")
+        sample_rate = 22050
+
+        with open(metadata_path, mode="w", newline='') as metadata_file:
+            metadata_writer = csv.writer(
+                metadata_file, delimiter="|", quoting=csv.QUOTE_NONE
+            )
+            for i, (transcript, normalized_transcript) in enumerate(
+                zip(cls.transcripts, cls.normalized_transcripts)
+            ):
+                fileid = "LJ001-000" + str(i)
```

Comment:
Can you do `f'LJ001-{i:04d}'`? That way, if someone adds more 6 new transcript examples, they still work.